CMakeLists.txt

```diff
@@ -1222,6 +1222,9 @@ if(BUILD_TEST)
   add_test(test_layout_op ${NVFUSER_ROOT}/tests/cpp/test_layout_op.cpp "")
   list(APPEND TEST_BINARIES test_layout_op)
 
+  add_test(test_mps ${NVFUSER_ROOT}/tests/cpp/test_mps.cpp "CUDA::cuda_driver")
```

Comment:
This is one example we should add a warning. We must not add more file than `test_mps.cpp`.